gfx/UltimateCataclysm/pngs_human_body_32x36/overlay/worn/belts_and_strapped_items/gartersheath.json

```diff
@@ -0,0 +1,22 @@
+[
+  {
+    "id": "overlay_female_worn_gartersheath1",
+    "fg": "overlay_female_worn_gartersheath1",
```

Comment:
Don't need this json. Don't need json when fg and id are the same, naming the files with the correct id is enough if you want to use the sprite for only one thing

Reply:
Does using json instead of outright naming it to the id is a better?